WireMockServer context manager fails to start server due to redirect

Environment:
- Python3.8
- MacOS Catalina
- requests==2.24.0
- wiremock==2.1.2

When I use `with WireMockServer` context manager, I am getting unexpected `WireMockServerNotStartedError` although server starts in the background (is visible in process list with e.g. `ps aux | grep java` and I can send requests to it).

From what I can tell, library makes a request to `/__admin` path but wiremock returns 302 Found and redirects to path with slash at the end: `/__admin/`. 

full stack trace:
```
self = <tests.<anonymized> object at 0x7f82f074a730>

    def test_(self):
>       with WireMockServer(port=49540) as wm:

03-integration-testing/03-04-client/tests/client/bik_verification_service_tests.py:17:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
<anonymized>/lib/python3.8/site-packages/wiremock/server/server.py:29: in __enter__
    self.start()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <wiremock.server.server.WireMockServer object at 0x7f82d0064a30>

    def start(self):
        if self.is_running:
            raise WireMockServerAlreadyStartedError("WireMockServer already started on port {}".format(self.port))

        cmd = [self.java_path, "-jar", self.jar_path, "--port", str(self.port), "--local-response-templating"]
        try:
            self.__subprocess = Popen(cmd, stdin=PIPE, stdout=PIPE, stderr=STDOUT)
        except OSError as e:
            raise WireMockServerNotStartedError(str(e))  # Problem with Java

        time.sleep(0.1)
        if self.__subprocess.poll() is not None:
            # Process complete - server not started
            raise WireMockServerNotStartedError(
                "\n".join(["returncode: {}".format(self.__subprocess.returncode), "stdout:", str(self.__subprocess.stdout.read())])
            )

        # Call the /__admin endpoint as a check for running state
        attempts = 0
        success = False
        while attempts < self.max_attempts:
            try:
                attempts += 1
                resp = requests.get("http://localhost:{}/__admin/".format(self.port))
                if resp.status_code == 200:
                    success = True
                    break
                time.sleep(0.25)
            except requests.exceptions.ConnectionError:
                pass

        if not success:
>           raise WireMockServerNotStartedError("unable to get a successful GET http://localhost:{}/__admin response".format(self.port))
E           wiremock.server.exceptions.WireMockServerNotStartedError: unable to get a successful GET http://localhost:49540/__admin response

<anonymized>/lib/python3.8/site-packages/wiremock/server/server.py:71: WireMockServerNotStartedError
```


I am willing to contribute to the project if this is unexpected behaviour (and not my mistake ;)) Otherwise, I am willing to contribute to docs but with some tips where to start, please ;)

--- a/wiremock/server/server.py
+++ b/wiremock/server/server.py
@@ -63,9 +63,10 @@ class WireMockServer(object):
                 if resp.status_code == 200:
                     success = True
                     break
-                time.sleep(0.25)
             except requests.exceptions.ConnectionError:
                 pass
+
+            time.sleep(0.25)
 
         if not success:
             raise WireMockServerNotStartedError("unable to get a successful GET http://localhost:{}/__admin response".format(self.port))

